src/actions/index.js

```diff
@@ -706,7 +706,10 @@ function getDatawolfHeader() {
 }
 
 export async function getRepoVersion() {
-	const versionRequest = await fetch("tags/github.json");
+	const versionRequest = await fetch("https://raw.githubusercontent.com/IN-CORE/IN-CORE/main/tags.json", {
+		method: "GET",
+		redirect: "follow"
+	});
 
 	try {
 		const githubVersions = await versionRequest.json();
```

Comment:
It might be add some default in the catch part incase github is down. Maybe return
```
{
"in-core": "NA",
  "incore-auth": "NA",
  "incore-docs": "NA",
  "incore-helm": "NA",
  "incore-lab": "NA",
  "incore-services": "NA",
  "incore-ui": "NA",
  "plotting-service": "NA",
  "pyincore": "NA",
  "pyincore-viz": "NA",
  "pyincore-data": "NA"
}
```
<img width="651" alt="image" src="https://user-images.githubusercontent.com/13950475/233410573-0c88c4cd-67f6-4ed4-b098-a48627f4b975.png">

Reply:
aahh, I see. Good idea. Thank you :D